modules/candidate_parameters/jsx/CandidateDOB.js

```diff
@@ -0,0 +1,235 @@
+  }
+
+  componentDidMount() {
+    let that = this;
+    $.ajax(
+      this.props.dataURL,
+      {
+        dataType: 'json',
+        success: function(data) {
+          let formData = {
+            dob: data.dob,
+          };
+
+           // Add parameter values to formData
+          Object.assign(formData, data.parameter_values);
+
+           that.setState({
+            Data: data,
+            isLoaded: true,
+            formData: formData,
+          });
+        },
+        error: function(data, errorCode, errorMsg) {
+          that.setState({
+            error: 'An error occurred when loading the form!',
+          });
+        },
+      }
+    );
+  }
+
+  setFormData(formElement, value) {
+    let formData = JSON.parse(JSON.stringify(this.state.formData));
+    formData[formElement] = value;
+
+     this.setState({
+      formData: formData,
+    });
+  }
+
+  onSubmit(e) {
+    e.preventDefault();
+  }
+
+  render() {
+    if (!this.state.isLoaded) {
+      if (this.state.error !== undefined) {
+        return (
+          <div className='alert alert-danger text-center'>
+            <strong>
+              {this.state.error}
+            </strong>
+          </div>
+        );
+      }
+
+       return (
+        <button className='btn-info has-spinner'>
+          Loading
+          <span
+            className='glyphicon glyphicon-refresh glyphicon-refresh-animate'>
+          </span>
+        </button>
+      );
+    }
+
+    let disabled = true;
+    let updateButton = null;
+    if (loris.userHasPermission('candidate_parameter_edit') &&
+      loris.userHasPermission('candidate_dob_edit')) {
+      disabled = false;
+      updateButton = <ButtonElement label="Update"/>;
+    }
+
+    let alertMessage = '';
+    let alertClass = 'alert text-center hide';
+    if (this.state.updateResult) {
+      if (this.state.updateResult === 'success') {
+        alertClass = 'alert alert-success text-center';
+        alertMessage = 'Update Successful!';
+      } else if (this.state.updateResult === 'error') {
+        let errorMessage = this.state.errorMessage;
+        alertClass = 'alert alert-danger text-center';
+        alertMessage = errorMessage ? errorMessage : 'Failed to update!';
+      }
+    }
+
+     return (
+      <div className='row'>
+        <div className={alertClass} role='alert' ref='alert-message'>
+          {alertMessage}
+        </div>
+        <FormElement
+          name='candidateDOB'
+          onSubmit={this.handleSubmit}
+          ref='form'
+          class='col-md-6'>
+          <StaticElement
+            label='PSCID'
+            text={this.state.Data.pscid}
+          />
+          <StaticElement
+            label='DCCID'
+            text={this.state.Data.candID}
+          />
+          <StaticElement
+            label='Disclaimer:'
+            text='Any changes to the date of birth requires an
+            administrator to run the fix_candidate_age script.'
+          />
```

Comment:
This is what it looks like now. Let me know if it's okay

<img width="824" alt="Screen Shot 2019-10-05 at 4 09 50 PM" src="https://user-images.githubusercontent.com/34260251/66270362-90f3b900-e820-11e9-8308-efb681c9247c.png">

Reply:
I'm not going to request the changes now, but after @nicolasbrossard's presentation on the server processes manager today it occurs to me that this would be a good place to use the SPM to launch the fix_candidate_age script asynchronously.